.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   rust-stable:
     docker:
-      - image: circleci/rust:latest
+      - image: cimg/rust:1.51.0
```

Comment:
Nit: this was a mistake in the other pr, I think using `1.51` is better so that we get fixes without having to change the version